plugins/inappauthplugin/plugin.cpp

```diff
@@ -57,8 +55,8 @@ Daemon::Plugins::InAppPlugin::beginUserInputInteraction(
                                   Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
                                   Q_ARG(QByteArray, QByteArray("example passphrase for unit test")));
         return Result(Result::Pending);
-    } else if (interactionParameters.promptText()
-            == QLatin1String("Enter the old master lock code for device secrets")) {
+    } else if (interactionParameters.promptText().endsWith(
+                   QLatin1String("Enter the old master passphrase."))) {
```

Comment:
Isn't the prompt text is already localized here?

Reply:
Yes, but this is only in the ifdef block for unit test builds.  Basically, this allows the plugin to automatically return some "user input" without actually requiring user interaction.  We should extend these cases to return data automatically for every unit test, to avoid the red-rectangle-prompt...